method-info/winsorisation/winsorisation.md

```diff
@@ -0,0 +1,423 @@
+# Winsorisation User Notes
+
+## Finding and Installing the Method
+
+You can find instructions on downloading and installing the method in the [Help centre](https://statisticalmethodslibrary.ons.gov.uk/help-centre/index) of the [ONS Statistical Methods Library](https://statisticalmethodslibrary.ons.gov.uk)
+
+## Using the Method
+
+### Overview
+
+The Winsorisation method is available as part of the statistical-methods-library package. It takes a dataframe as input that contains the data to be considered for Wisorisation. This method implements one-sided Winsorisation only.
+
+#### Input Data
+
+An example dataset and a walkthrough of the processing is available in a subsequent section.
+
+The input dataframe comprises of rows of data that must include the following fields:
+
+* Reference - Any - a unique identifier, e.g Business Reporting Unit
+* Period - String - for example "YYYY" or "YYYYMM", the period to be Winsorised
+* Cell or Group - Numeric - a number representing a stratum (e.g Standard
+Industrial Classification)
+* Target Variable - Numeric - the value to be treated
+* L-value - Numeric - a pre-calculated value closely associated with the data to be processed
+* Design Weight - Numeric - a supplied weight that
+reflects the sampling design
+* Calibration Factor - Numeric - Nulls allowed, a weight that maintains the
+estimated calibration totals. Mandatory for ratio estimation.
+* Auxiliary - Numeric - Nulls allowed, a secondary value used for prediction
+in ratio estimation. Mandatory for ratio estimation.
+
+The presence of Auxiliary and Calibration Factor indicates that ratio
+estimation must be performed rather than expansion estimation.
+
+#### Output
+
+The Winsorisation method returns a marker indicating whether a given row of data has been used for Winsorisation or not along with an outlier weight which indicates the reduction in the associated target variable due to Winsorisation.
+
+* Reference - Any - a unique identifier, e.g Business Reporting Unit
+* Period - String - for example "YYYY" or "YYYYMM", the period to be Winsorised
+* Outlier Weight - Numeric - A value between 0 and 1 which reflects the
+reduction in a target variable due to Winsorisation
+* Marker - String - Indicates the result of the Winsorisation method for
+each target value
+  * **W** - Winsorised. Marked against any value that could be Winsorised
+  * **NW_FE** - Not Winsorised Fully Enumerated. Marked when design weight is
+  1 for expansion estimation.
+  * **NW_AG** - Not Winsorised a*g. Marked
+  in ratio estimation when the product of
+  design weight and calibration factor is <= 1.
+
+The methodological specification avaialble at [https://statisticalmethodslibrary.ons.gov.uk/](https://statisticalmethodslibrary.ons.gov.uk/) contains a detailed breakdown of the mathematics used in Winsorisation.
```

Comment:
* available

may use the word 'here' as a link label

Reply:
Changed the typo on the available. Not sure if you had a comment about the link, I have purposefully left it pointing to the URL of the website to make it clear there is a supporting website.